### ISSUE TITLE
Issue with the "last played" counter

### ISSUE THREAD

**Issue by pajarorrojo** [batocera-linux/batocera-emulationstation#1754]
### Concerned project

Batocera

### Distribution Version

Batocera V40 2024/06/28

### Architecture

Steam Deck

### Issue description

Hi @fabricecaruso i hope you are doing well... a little issue here:
The "last played" counter always shows 16 hours when the game has just been launched.

The line of code in the theme is:
` <text>'' + '${theme-lastplayedLabel}' + '... ' + {game:lastplayed}.elapsed().proper() + ''</text>`

https://github.com/pajarorrojo/es-theme-PlayStation-X/blob/7ab34f3270c7e6368172b4a4c00d7ee17e8de904/_theme_views/top-info.xml#L290

Thanks!

![screenshot-2024 07 06-16h52 07](https://github.com/batocera-linux/batocera-emulationstation/assets/73712202/cb84aa1a-93a3-458d-8258-c0f55ca25e81)

![screenshot-2024 07 06-16h51 52](https://github.com/batocera-linux/batocera-emulationstation/assets/73712202/7964fcdb-f851-4e02-8f7c-5ed2866e8632)

**Comment by pajarorrojo** [comment]
**UPDATE:** I've noticed that the data it shows is the hours since the start of the day... For example, if I run the game at 8pm (20.00h), it shows: 20 hours ago. If I run it at 4pm it shows 16 hours ago... etc.
![screenshot-2024 09 18-21h48](https://github.com/user-attachments/assets/24b666af-b9bb-4c45-93cc-579f8aed8f51)

**Comment by Snapouille** [comment]
I agree that we're dealing with a bug, I have exactly the same problem.

**Comment by Darknior** [comment]
Mais c'est un nouveau bug ? de la v40 ou tu ne l'avais jamais vue avant ?
J'avoue je n'ai pas fais attention, mais si c'est le cas c'est moche ... de mémoire avant ça fonctionnait pour moi. Genre en v39 ...

**Comment by Snapouille** [comment]
@Darknior yes it's a new bug, before it worked fine

**Comment by pajarorrojo** [comment]
@Snapouille @Darknior This is a bug since v40... note that my first post is from July. Maybe not many people have noticed why this happens with the new variable {game:lastplayed} and not with the old md_lastplayed label name. Not many themes are updated with the new bindings variables.

**Comment by pajarorrojo** [comment]
Technically, its a bug since variable bindings have been enabled, it’s just that most themes aren’t using them yet

**Comment by fabricecaruso** [comment]
Should be solved with https://github.com/batocera-linux/batocera-emulationstation/pull/1916
